### PR TITLE
DOCS: Add link to cmake download page

### PR DESCRIPTION
**Pull request by bkmgit**
Add a link to the cmake download page where various options for installing cmake can be found.

### CODE REVIEW

File: docs/dev/build-instructions/build_linux.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ The dependence on `g++-multilib` only applies if you are on a 64bit system and w
 If you are using a distribution that is not based on Ubuntu, you probably have to change the command accordingly.
 
 You will need `cmake` **v3.15 or later**. If the version shipped by your distribution is not recent enough, you can install a recent one via the
-[official PPA](https://apt.kitware.com/) by following the instructions on the linked page.
+[official PPA](https://apt.kitware.com/) or by from the [linked page](https://cmake.org/download/).
```
Comment (davidebeatrici) [inline docs/dev/build-instructions/build_linux.md:43]:
```suggestion
[official PPA](https://apt.kitware.com/) or from the [linked page](https://cmake.org/download/).
```

Comment (bkmgit) [reply docs/dev/build-instructions/build_linux.md]: Thanks.

**Review by Krzmbrzl — CHANGES_REQUESTED** [review]
Please squash the 2 commits. After that this LGTM

**Comment by bkmgit** [comment]
@Krzmbrzl  Assuming no other changes are needed, is it possible to squash commits on merging:
https://github.blog/2016-04-01-squash-your-commits/

**Comment by davidebeatrici** [comment]
It's possible, but it would not create a merge commit.

**Comment by bkmgit** [comment]
Thanks. Updated. Is it ok now?

**Review by Krzmbrzl — APPROVED** [review]

**Pull request merged**